.github/workflows/continuous-integration.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         if: "${{ matrix.extension == 'sqlite3' }}"
 
       - name: "Run PHPUnit"
-        run: "vendor/bin/phpunit -c ci/github/phpunit/${{ matrix.extension }}.xml --coverage-clover=coverage.xml"
+        run: "vendor/bin/phpunit -c ci/github/phpunit/${{ matrix.extension }}.xml --coverage-clover=coverage.xml --display-warnings"
```

Comment:
Is this equivalent to using `displayDetailsOnTestsThatTriggerNotices` in our configuration files? I'm asking because if it is, then we might want to do that inside `phpunit.xml` so that there isn't a situation where contributors start to think they need to push to Github to get details about a warning. Also, I'm noticing that similar settings exist for notices, errors and deprecations. Shouldn't we turn them on as well?

Reply:
I don't know. I could also revert the commit that adds this option. I needed it to debug the warnings, but they're fixed now.

Reply:
Dropping that commit sounds good as well 👍
We can think about configuration more globally in a separate issue.